packages/sanity/src/structure/comments/src/components/list/CommentsListItemLayout.tsx

```diff
@@ -298,28 +310,58 @@ export function CommentsListItemLayout(props: CommentsListItemLayoutProps) {
         <Flex align="center" gap={FLEX_GAP} flex={1}>
           <CommentsAvatar user={user} />
 
-          <Flex align="center" paddingBottom={1} sizing="border" flex={1}>
-            <Flex align="flex-end" gap={2}>
-              <Box flex={1}>{name}</Box>
-
-              {!displayError && (
-                <Flex align="center" gap={1}>
-                  <TimeText muted size={0}>
-                    <time dateTime={createdDate.toISOString()} title={formattedCreatedAt}>
-                      {createdTimeAgo}
-                    </time>
-                  </TimeText>
-
-                  {formattedLastEditAt && editedDate && (
-                    <TimeText muted size={0} title={formattedLastEditAt}>
-                      <time dateTime={editedDate.toISOString()} title={formattedLastEditAt}>
-                        ({t('list-item.layout-edited')})
+          <Flex direction="column" gap={2} paddingY={comment.context?.intent ? 2 : 0}>
+            <Flex
+              align="center"
+              paddingBottom={comment.context?.intent ? 0 : 1}
+              sizing="border"
+              flex={1}
+            >
+              <Flex align="flex-end" gap={2}>
+                <Box flex={1}>{name}</Box>
+
+                {!displayError && (
+                  <Flex align="center" gap={1}>
+                    <TimeText muted size={0}>
+                      <time dateTime={createdDate.toISOString()} title={formattedCreatedAt}>
+                        {createdTimeAgo}
                       </time>
                     </TimeText>
-                  )}
-                </Flex>
-              )}
+
+                    {formattedLastEditAt && editedDate && (
+                      <TimeText muted size={0} title={formattedLastEditAt}>
+                        <time dateTime={editedDate.toISOString()} title={formattedLastEditAt}>
+                          ({t('list-item.layout-edited')})
+                        </time>
+                      </TimeText>
+                    )}
+                  </Flex>
+                )}
+              </Flex>
             </Flex>
+
+            {comment.context?.intent && (
```

Comment:
Do we want to show the intent link for all replies – or would it be sufficient to show it only on the root comment?

<img width="867" alt="Screenshot 2024-02-12 at 18 11 45" src="https://github.com/sanity-io/sanity/assets/15094168/eedb5486-e0a5-48d1-8173-b18ad3252e52">

Reply:
Hmm, it's a good question, I don't love the left side visually. I suppose it only serves a purpose if a response needs to show some different context, say to contrast how some content looks across two different pages.

Should we hide unless it differs from the root context?

Reply:
Kind of like this?

<img width="627" alt="Screenshot 2024-02-12 at 20 40 38" src="https://github.com/sanity-io/sanity/assets/8467307/2ba9509d-fdcf-4c31-ac17-92d8f9f7955f">

I've made the necessary changes on a local stash, so ready to push up if we want to do this.

Reply:
I might miss some things here – but is it currently possible to add a reply with a different context than the root comment? How would you do that?

Reply:
The Presentation tool tracks the URL the preview pane is displaying, so for example a document of type `shoe` could be populating two pages: a listing page `/shoes` and an item page `/shoes/slug`.

It's probably not typical, but you might have e.g. a comment chain about the size of the title. You could make a single comments thread about how the title looks on each page, with intent links to each context.

Reply:
Thanks for clarifying! Then I think your suggestion makes sense, that is, to hide the intent link unless it differs from the root context 👍

Reply:
Great! Added in https://github.com/sanity-io/sanity/pull/5576/commits/30b068a1b463e62252f884a6711cd2438dcf35f8.

Reply:
Thanks!

Another thought (no block for merge): should we check that Presentation is being used to determine if the intent link should be displayed? Or maybe by checking the presence of the intent context provider? So that we know for sure that the intent link actually has a target in the Studio.

Reply:
Yes that definitely sounds like a good idea, I'll create a Linear issue for it.

Happy to resolve this thread?